examples/safety_and_security/retail_agent/README.md

```
@@ -23,790 +23,430 @@ limitations under the License.
+PIIDefenseMiddleware: Verified output of <workflow>: No PII detected
+... omitted for brevity
+```
+
+### Before vs After Comparison
 
 <table>
   <tr>
     <td align="center">
       <b>Before Defenses</b><br/>
-      <img src="./src/nat_retail_agent/data/attack-score.png"/>
+      <img src="./assets/attack-score.png"/>
```

markdownlint-cli2 (0.18.1)

Images should have alternate text (alt text)

(MD045, no-alt-text)